src/commands/configuration/config.js

```diff
@@ -92,3 +94,31 @@ class ConfigPrefix extends Command {
     channel.send(embed)
   }
 }
+
+class ConfigDJRole extends Command {
+  constructor (client, parentCommand) {
+    super(client, parentCommand)
+    this.name = 'djrole'
+
+    this.parameters = new CommandParameters(this,
+      new StringParameter({ full: true, required: false, missingError: 'errors:invalidRole' })
+    )
+  }
+
+  async run ({ t, author, channel, guildDocument }, role) {
+    const regexResult = await ROLE_REGEX.exec(role)
+    let roleId = regexResult[0]
+    const embed = new SwitchbladeEmbed(author)
+    const guildrole = await channel.guild.roles.find(r => r.id === roleId)

```

Comment:
you *could* also search for the role name, just sayin'

Reply:
we are not searching for role name

Reply:
bUt YoU cAn, FoR eAsE oF uSe

Reply:
> bUt YoU cAn, FoR eAsE oF uSe

eu falei isso pro pedro, ele falou que nao, acabou